canonical-websites/www.ubuntu.com
# Update ja UA service description

## Done
Updated the JA UA service description and archived the old one.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja)
- Check it looks ok

## Issue / Card
Fixes https://github.com/canonical-websites/jp.ubuntu.com/issues/23
